.github/workflows/release.yml

```diff
@@ -1,16 +1,32 @@
-name: release
+name: Release
 
 on:
-  push:
-    tags:
-      - "v*"
+  workflow_dispatch:
+    inputs:
+      release:
+        description: "The New Release"
+        required: true
+
+env:
+  RELEASE: ${{ github.event.inputs.release }}
 
 permissions:
   contents: write
 
 jobs:
   release:
     runs-on: ubuntu-latest
+
     steps:
       - uses: actions/checkout@v2
-      - uses: cli/gh-extension-precompile@v1
+
+      - name: Setup Go
+        uses: actions/setup-go@v2.1.4
+        with:
+          go-version: "1.17.3"
+
+      - name: Release
+        run: |
+          echo ${{ github.token }} | gh auth login --with-token
```

Comment:
I don't think you need this since `gh` should automatically pick up the GITHUB_TOKEN
```suggestion
```

Reply:
👍

Reply:
done

Reply:
I make `github.token` as a env variable